ray-operator/controllers/ray/common/pod.go

```diff
@@ -33,10 +33,13 @@ const (
 	// If set to true, kuberay auto injects an init container waiting for ray GCS.
 	// If false, you will need to inject your own init container to ensure ray GCS is up before the ray workers start.
 	EnableInitContainerInjectionEnvKey = "ENABLE_INIT_CONTAINER_INJECTION"
-	NeuronCoreContainerResourceName    = "aws.amazon.com/neuroncore"
-	NeuronCoreRayResourceName          = "neuron_cores"
-	TPUContainerResourceName           = "google.com/tpu"
-	TPURayResourceName                 = "TPU"
+
+	// EnableLoginBashEnvKey is the environment variable to enable login bash for Ray containers.
```

Comment:
"This environment variable for the KubeRay operator determines whether to enable a login shell by passing the `-l` option to the container command `/bin/bash`. The `-l` flag was added by default before KubeRay v1.4.0, but it is no longer added by default starting with v1.4.0."